examples/inverse/plot_covariance_whitening_dspm.py

```diff
@@ -48,12 +48,12 @@
 data_path = spm_face.data_path()
 subjects_dir = data_path + '/subjects'
 
-raw_fname = data_path + '/MEG/spm/SPM_CTF_MEG_example_faces%d_3D_raw.fif'
+raw_fname = data_path + '/MEG/spm/SPM_CTF_MEG_example_faces%d_3D.ds'
 
-raw = io.Raw(raw_fname % 1)  # Take first run
+raw = io.read_raw_ctf(raw_fname % 1, preload=True)  # Take first run
```

Comment:
Whoops. I don't know how that got there. But in any case there is a ``load_data`` on the line below so this seems redundant.